### PR TITLE
fix(dropdown-select): disabled state for chromium-based browsers

### PR DESCRIPTION
**Pull request by BY00565233**
## Description
Broken styling of disabled Dropdown Select in Chromium-based browsers

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2081

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: src/scss/components/dropdown-select/index.scss [hunk 1]
```diff
@@ -73,6 +73,10 @@ hx-select-control {
   }
 }
 
+#selLongOption-disabled {
+  opacity: 1;
+}
+
```
Comment (100stacks) [inline src/scss/components/dropdown-select/index.scss:79]: We should not need to add an `id` special case here.  Please update the  **`disabled`** mixins.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/dropdown-select/index.scss (hunk 1 above)
Comment (BY00565233) [reply src/scss/components/dropdown-select/index.scss]: Changes have been done, Please review

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**